social-feed/App.tsx

```diff
@@ -257,34 +239,35 @@ const App = () => {
       };
     },
   );
-
   return (
     <KeyboardAvoidingView
       behavior={Platform.OS === 'ios' ? 'padding' : 'height'}
       style={{flex: 1}}>
       <GestureHandlerRootView style={{flex: 1}}>
         <NavigationContainer ref={navigationRef} independent={true}>
-        {apiKey && myClient ? (
-          <LMOverlayProvider
-            myClient={myClient}
-            apiKey={apiKey}
-            userName={userName}
-            userUniqueId={userUniqueID}
-            lmFeedInterface={lmFeedInterface}
-            callbackClass={callbackClass}
-            isUserOnboardingRequired={true}
-            >
+          {apiKey && myClient ? (
+            <LMOverlayProvider
+              myClient={myClient}
+              apiKey={apiKey}
+              userName={userName}
+              userUniqueId={userUniqueID}
+              lmFeedInterface={lmFeedInterface}
+              callbackClass={callbackClass}
+              isUserOnboardingRequired={true}>
               <Stack.Navigator screenOptions={{headerShown: false}}>
-              <Stack.Screen
-                name={USER_ONBOARDING_SCREEN}
-                component={LMUserOnboardingScreen}
-                options={{
-                  headerShown: false,
-                }}
+                <Stack.Screen
+                  name={USER_ONBOARDING_SCREEN}
+                  component={LMUserOnboardingScreen}
+                  options={{
+                    headerShown: false,
+                  }}
                 />
                 <Stack.Screen
                   name={UNIVERSAL_FEED}
                   component={LMSocialFeedScreen}
+                  initialParams={{
+                    personalisedFeed: FeedType.PERSONALISED_FEED,
```

Comment:
```suggestion
                    feedType: FeedType.PERSONALISED_FEED,
```
param name should be `feedType`